back-office/advanced-parameters/multistore/multistore.md

```diff
@@ -0,0 +1,65 @@
+# SPECS TO SETUP THE MULTISTORE
+
+As a merchant I need to be able to create many stores
+
+## WHAT
+
+Thus, this project aims at specifying the current and expected behaviour to setup many stores
+
+## SPECIFICATIONS
+
+### Enable the multistore
+
+The multistore is disabled by default and can be enabled in Shop parameters > General <br/>
+Once enabled, a new "Multistore" menu is displayed in Advanced parameters > Multistore
+
+### Shop group
+
+A shop group can contains many stores. <br/>
```

Comment:
can contain

Reply:
Before talking in details about shop group and shop, I think it's good to have a quick overview of what you can do. 
Ex : "There are two types of main actions in this page : 
- create a new shop - to be able to have different shops administered by the same back office
- create a group of shops - to be able to... 

(c'est juste un exemple, mais c'est pour commencer par la vue d'ensemble et après rentrer dans le détail)